eng/testing/performance/performance-setup.ps1

```diff
@@ -125,6 +125,10 @@ if ($RunFromPerformanceRepo) {
 }
 else {
```

Comment:
For the sh, it appears that lines 275, 281, 288, 290, 302, 307, 313, and 318 need such a check. Is that the case for all of them? Did I miss anything?